pages/common/syncthing.md

```diff
@@ -7,6 +7,10 @@
 
 `syncthing`
 
+- Start syncthing without opening a browser:
```

Comment:
Perhaps we should be slightly more specific here?
```suggestion
- Start syncthing without opening a web browser:
```

Reply:
Absolutely agree, done!